pyvista/core/filters/data_object.py

```diff
@@ -2753,3 +2789,169 @@ def sample(  # type: ignore[misc]
                 raise VTKVersionError(msg)
         _update_alg(alg, progress_bar, 'Resampling array Data from a Passed Mesh onto Mesh')
         return _get_output(alg)
+
+    def cell_quality(  # type: ignore[misc]
+        self: _DataSetOrMultiBlockType,
+        quality_measure: Literal['all', 'all_valid']
+        | _CellQualityLiteral
+        | Sequence[_CellQualityLiteral] = 'scaled_jacobian',
+        *,
+        null_value: float = -1.0,
+        progress_bar: bool = False,
+    ) -> _DataSetOrMultiBlockType:
+        r"""Compute a function of (geometric) quality for each cell of a mesh.
+
+        The per-cell quality is added to the mesh's cell data, in an array with
+        the same name as the quality measure. Cell types not supported by this
+        filter or undefined quality of supported cell types will have an
+        entry of ``-1``. See table below for all measures and the
+        :class:`~pyvista.CellType` supported by each one.
+
+        Defaults to computing the scaled jacobian quality measure.
+
+        .. dropdown:: Cell Quality Measures
```

Comment:
This table is essential to the documentation of this filter, I think we should use the `open` option to have it be 'dropped down' on open.